group/add-member/v2/schema.js

```diff
@@ -32,6 +33,11 @@ module.exports = {
       pattern: '^v2$'
     },
     secret: { $ref: '#/definitions/secret' },
+    oldSecrets: {
+      type: 'array',
+      items: [{ $ref: '#/definitions/secret' }],
+      minItems: 0
+    },
```

Comment:
might have to add some special logic in tribes2 for this though which would mean more coding

Reply:
not a bad idea... but would rather not right now - the problem is that the `group/init` doesn't necessarily know about all past epochs.... oh, well it does actually, just not the parallel ones.

There might be a problem with how many keys you could add to a new init.
I kinda like the idea the more I think about it.

Reply:
Since this is coded already and it's not too bad I'm gonna put this in this "stuff we could maybe do if we have more time" issue https://github.com/ssbc/ssb-group-exclusion-spec/issues/30 and just merge